src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -35,4 +38,27 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'net5.0'">
+    <!--Remove the Linux-specific code-->
+    <Compile Remove="System\Device\Gpio\Drivers\Windows10Driver.notSupported.cs" />
+    <Compile Remove="System\Device\I2c\I2cDevice.nonWindows.cs" />
+    <Compile Remove="System\Device\Pwm\PwmChannel.nonWindows.cs" />
+    <Compile Remove="System\Device\Spi\SpiDevice.nonWindows.cs" />
+    <Compile Remove="System\Device\CommonHelpers.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
+    <!--Remove Windows Code-->
+    <Compile Remove="Interop\Windows\WinRT\*.cs" />
```

Comment:
I think the PR lacks files. This directory is empty for me (except for "Interop.WaitForCompletion.cs", which only contains a helper method). 

Besides, I don't understand why "net5.0" or not would have anything to do with Windows or not.

Reply:
> I think the PR lacks files. This directory is empty for me (except for "Interop.WaitForCompletion.cs", which only contains a helper method).

I know this is the only WinRT file today, but I used wildcards as any file that goes in it in the future would have to be removed, and because everything is added to all configurations by default, I wanted to make sure that we guarded against new files or renames.

> Besides, I don't understand why "net5.0" or not would have anything to do with Windows or not.

Our net5.0 configuration is effectively a non-Windows build since we want everyone targeting net5.0 and Windows to use the net5.0-windows version instead. That is why net5.0 version now throws anytime you are trying to use anything that relies on WinRT code.